salt/preflight/defaults.yaml

```diff
@@ -0,0 +1,32 @@
+destructive: False      # define if /var/lib/etcd if should be removed
+upgrade: False          # define if we're on an update case
+mandatory:
+  packages:
+    - util-linux            # provides nsenter, mount
+    - iproute               # provides ip
+    - iptables              # provides iptables
+  sysctl_values:
+    net.bridge.bridge-nf-call-ip6tables: 1
+    net.bridge.bridge-nf-call-iptables: 1
+    net.ipv4.ip_forward: 1
+  available_pathdirs:
+    - /var/lib/etcd
+    - /etc/kubernetes/manifest
+  services:
+    running:
+      - containerd
+  ports:
+    - 10250
+    - 10251
+    - 10252
+    - 2379
+recommended:
+  packages:
+    - ebtables              # provides ebtables
+    - ethtool               # provides ethtool
+    - socat                 # provides socat
+    - iproute               # provides tc
+    - coreutils             # provides touch
+  services:
+    running:
+      - kubelet
```

Comment:
Oh? This is surprising... I thought there's a check in `kubeadm` to ensure it's *not* running. Also, given the requirement for port 10250 to be available, this can't be true.

Reply:
Right, I'll change it into service.enabled:
[warning] if kubelet service does not exist or if it is disabled